halconfig/settings.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+var feedbackUrl = "";
```

Comment:
Not a big deal, but the choice of hard-coded variables (`feedbackUrl`, `entityTagsEnabled`, `netflixMode`) feels arbitrary.